rasa/core/run.py

```diff
@@ -74,6 +74,23 @@ def _create_app_without_api(cors: Optional[Union[Text, List[Text]]] = None):
     return app
 
 
+def _create_ssl_context(
+    ssl_certificate: Optional[Text], ssl_keyfile: Optional[Text], ssl_password: Text
+) -> Optional["SSLContext"]:
+    """Create a SSL context (for the sanic server) if a proper certificate is passed."""
+
+    if ssl_certificate:
+        import ssl
+
+        ssl_context = ssl.create_default_context(purpose=ssl.Purpose.CLIENT_AUTH)
+        ssl_context.load_cert_chain(
+            ssl_certificate, keyfile=ssl_keyfile, password=ssl_password
```

Comment:
[here](https://docs.python.org/3/library/ssl.html) password is an optional arg with default `None` but we pass `""`, is that fine?

Reply:
yes good point - I looked in the incode docs and from there it looked like `""` is the default. doesn't really matter as it is not used if no PW is required. changed it though to `None` as it is cleaner :+1: